docs/source/conf.py

```diff
@@ -145,6 +146,18 @@
 # If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
 html_show_copyright = True
 
+
+
+# -- Options for Sphinx Gallery ----------------------------------------------
+
+sphinx_gallery_conf = {
+    "examples_dirs": ["../../modopt/examples/"],
+    "filename_pattern": "/example_",
+    "ignore_pattern": r"/(__init__|conftest)\.py",
+}
```

Comment:
You can see some of the options I set for PySAP [here](https://github.com/CEA-COSMIC/pysap/blob/develop/docs/source/conf.py#L85). Feel free to play around, I can always update PySAP to match the configuration to set here.